src/include/network/distributed/logging_service.pb.h

```diff
@@ -42,7 +42,7 @@
 // @@protoc_insertion_point(includes)
 
 namespace peloton {
-namespace networking {
+namespace distributed {
```

Comment:
Instead of `peloton::distributed`, I think it should be `peloton::networking::service`.